share/completions/git.fish

```diff
@@ -856,8 +856,8 @@ complete -c git -n '__fish_git_using_command diff show range-diff log' -l pickax
 complete -c git -n '__fish_git_using_command diff show range-diff log' -l pickaxe-regex -d 'Treat the <string> given to -S as an extended POSIX regular expression to match'
 complete -c git -n '__fish_git_using_command diff show range-diff log' -l relative -d 'Exclude changes outside the directory and show relative pathnames'
```

Comment:
can you rebase on master instead of merging into your branch? that would simplify the history

Reply:
Done 👍